tests/integration/kubernetes/confidential_kbs.sh

```diff
@@ -139,7 +139,7 @@ kbs_install_cli() {
 	# Mininum required version to build the client (read from versions.yaml)
 	local rust_version
 	ensure_yq
-	rust_version=$(get_from_kata_deps "externals.coco-kbs.toolchain")
+	rust_version=$(get_from_kata_deps "externals.coco-trustee.toolchain")
```

Comment:
The change is fine, but the commit message is really cryptic. How about `externals.coco-kbs.toolchain` is not defined, get the rust_version from `externals.coco-trustee.toolchain`?

Reply:
Ditto!